README.md

```diff
@@ -1,5 +1,82 @@
 # MongoDB Realm CLI
 
+## Building the CLI
+
+The CLI can be run many different ways.  To install it globally on your machine, you can use `npm` to do so:
+
+```cmd
+npm install -g mongodb-realm-cli
+```
+
+### Running from Source
+
+You may wish to run the CLI from source, in which case `go` provides a few different options.
+
+#### Using `go run`
+
+The entry point for the CLI is in `main.go`, so simply running `go run main.go` from the root of the project repo is equivalent to invoking the installed command `realm-cli`.  It accepts commands and flags as normal.
+
+The one caveat here is you may wish to be running the tool from a clean directory, so as not to write app configuration files to the project repo.  However, for commands that do not interact with the local filesystem, this is often the quickest and easiest way to run from source.
+
+#### Using `go build`
```

Comment:
potentially worth adding some instructions for `go install` here as well. I personally find it more convenient to use for local development vs. `go build`, since `go install` will automatically build the binary and put it in `~/go/bin`, which most people will presumably already have added to their path.

Reply:
Yeah that’s a good call out. It crossed my mind initially but I’ve frankly yet to use it that way locally, so I didn’t have a confident/confirmed command to copy/paste (I’m hoping the -ldflags still works the same?) I’ll play around with it and get something for that though